src/Microsoft.DncEng.SecretManager/SecretTypes/ZenHubAccessToken.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text.RegularExpressions;
+using Microsoft.DncEng.CommandLineLib;
+
+namespace Microsoft.DncEng.SecretManager.SecretTypes
+{
+    [Name("zenhub-access-token")]
+    public class ZenHubAccessToken : GenericAccessToken
+    {
+        private readonly string[] _expirationDateFormats = new[] { "yyyy-MM-dd", "yyyy-MM-dd HH:mm:ss" };
+
+        protected override string HelpMessage => "Please login to https://app.zenhub.com/dashboard/tokens using your GitHub account and create a new token.";
```

Comment:
Does the user need group membership or other stuff for their GH account to do this?  May be worth mentioning.

Reply:
I do not know. They do need to be able to use zenhub for what we need. So I assume just "be in our team"